frontend/src/components/profile/ProfileEdit.jsx

```diff
@@ -1,18 +1,69 @@
+import axios from "axios";
 import { Button, Paper, TextField } from "@mui/material";
-import React, { useRef, useState } from "react";
-import { useNavigate } from "react-router-dom";
+import React, { useRef, useState, useEffect } from "react";
+import { useNavigate, useParams } from "react-router-dom";
 import { StyledForm } from "../StyledComponents.jsx";
+import { Cookies } from "react-cookie";
 
 function ProfileEdit() {
```

Comment:
We should prevent users that are not signed in from accessing this page. Could probably just check if the token exists and route them to sign in if not.
nit: change the styling of submit button to match theme